packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -847,13 +847,14 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
         label={item.text}
         title={title}
         // eslint-disable-next-line react/jsx-no-bind
-        onRenderLabel={this._onRenderItemLabel.bind(this, item)}
+        onRenderLabel={this._onRenderItemLabel.bind(this, item, this._listId + item.index + '-label')}
```

Comment:
(nit) maybe save this in `const labelId = this._listId + item.index + '-label';` since it's also used for ariaLabelledBy below
```suggestion
        onRenderLabel={this._onRenderItemLabel.bind(this, item, labelId)}
```

Reply:
Updated to save this + the `this._listId + item.index` value, since that's also used in multiple places 👍